packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit_api_impls.dart

```diff
@@ -481,7 +481,7 @@ class WKHttpCookieStoreHostApiImpl extends WKHttpCookieStoreHostApi {
   }
 
   /// Calls [setCookie] with the ids of the provided object instances.
-  Future<void> setCookieForInsances(
+  Future<void> setCookieForInstances(
```

Comment:
This is technically public API, so we might need to do a breaking change for this. 
https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#breaking-changes

Reply:
The plugin doesn't expose these classes: https://github.com/flutter/plugins/blob/main/packages/webview_flutter/webview_flutter_wkwebview/lib/webview_flutter_wkwebview.dart. 

However, Dart does still allow access to implementation files. But, there are lint warning about importing implementation files of a package and proper usage of the plugin will make accessing them difficult.

Reply:
Yeah, technically one can use those classes by importing `webview_flutter_wkweview/lib/src/web_kit/web_kit_api.impls.dart`, and those app will break after this change.

I'm not sure the policy of making "public" API change in these files.

Reply:
cc @stuartmorgan Is there documented policy on changing files in `lib/src` in a plugin?

In Dart's documentation on implementation files, it looks to be implied that changing files in `lib/src` are not considered a breaking change. As a user shouldn't be importing them: https://dart.dev/tools/pub/package-layout#implementation-files

Reply:
Anything we're not exporting is fair game to change.